### ISSUE TITLE
HelpViewer 2.3 and VS2019

### ISSUE THREAD

**Issue by kfertitta** [EWSoftware/SHFB#708]
It seems that VS2019 is the first VS version to use the same Help Viewer version as an earlier VS version (VS2017), unless I"m mistaken.

Given this, does that mean, on a machine with both VS2017 and VS2019 installed (perhaps multiple instances of each), we invoke the Help Viewer 2.3 twice -- specifying different catalog names?  Seems odd, but maybe that's correct.  If we only need invoke it once (since it's the same Help Viewer version), then which catalog name do we supply?

Thanks in advance.

**Comment by EWSoftware** [comment]
I just did a quick test by building a Help Viewer file in VS 2019 and installing it using the current SHFB release.  It didn't seem to make any difference using the 2017 catalog name.  The help was available in the viewer and looking up a member from the editor in the IDE opened the help viewer and displayed the proper topic.

**Comment by kfertitta** [comment]
Hi Eric,

Thanks for the reply.  Yeah, I did some experimentation myself several days ago and hadn't replied here, but my results are the same as yours.  I tested several configurations -- a VM with only VS2017, a VM with only VS2019, a VM with both VS2017 and VS2019, and a VM with multiple instances of both VS2017 and VS2019.  In all cases, just supplying the VS2017 catalog name worked.

Even though there is only a single help viewer instance installed, I did note that, in cases where you have multiple instances of the same VS version (say, Community + Pro + Enterprise), you still had to explicitly install the Help Viewer in each using the VS installer.  Installing into one did not make the View Help command available in any of the others.  They must just be doing some ref-count or something within the installer in each VS silo.
